interface/patient_file/encounter/forms.php

```diff
@@ -667,10 +667,18 @@ function myGetRegistered($state="1", $limit="unlimited", $offset="0") {
     if ($result['squad'] && ! acl_check('squads', $result['squad'])) {
         $pass_sens_squad = false;
     }
+
+    //fetch acl for category of given encounter
+    $pc_catid = fetchCategoryIdByEncounter($encounter);
+    $postCalendarCategoryACO = fetchPostCalendarCategoryACO($pc_catid);
+    $postCalendarCategoryACO = explode('|',$postCalendarCategoryACO);
+    $authPostCalendarCategory = acl_check($postCalendarCategoryACO[0], $postCalendarCategoryACO[1]);
```

Comment:
updated code so if no aco is set there is authorization (like in form pernissions if no aco is set for a form in registry)